docs/guides.rst

```diff
@@ -130,14 +130,19 @@ For example, to change the timestep, create a ``SomdConfig`` object and pass it
 
 .. code-block:: python
 
-    engine_config = a3.SomdConfig(timestep=2.0) # Create configuration with custom parameters
+    engine_config = a3.SomdConfig(timestep=2.0) # fs (femtoseconds), Create configuration with custom parameters
     calc = a3.Calculation(engine_config=engine_config) # Pass to Calculation during creation
     
     # Or modify parameters after creating the Calculation
     calc = a3.Calculation()
-    calc.engine_config.timestep = 2.0
+    # Works if the calculation has not been setup yet
+    calc.engine_config.timestep = 2.0 # fs
```

Comment:
Thanks very much for pointing this out in the docs - this is a super important point. I feel like this may still cause some confusion if people don't read the docs properly. I wonder if we could warn the user if they edit the config directly when the calc has already been setup? Do you have any ideas? Don't worry if not as it's already way better than it used to be! This is fine, just trying to minimise confusion. Might be worth reminding users why this is required - just saying because the configs for the legs etc also need updated